client/src/pages/singlePlace/index.js

```diff
@@ -0,0 +1,175 @@
+    preservation: [
+      {
+        title: 'hi',
+        badges: ['asdjlakfsjlksaj', 'sadsadasd'],
+      },
+    ],
+  },
+    ],
+  };
+
+  render() {
+    const { name, country, city, website, images } = this.props;
+    const { reviews } = this.state;
+    return (
+      <>
+        <img
+          src={backgroundImage}
+          alt="background"
+          className="singlePlace-header-img"
+        />
+        <div className="buttons-container">
+          <Link to= "/search"><span className="buttons-container__back">
+            <Icon type="arrow-left" />
+            <span className="buttons-container__back__text">
+              Back to results
+            </span>
+          </span></Link>
+          <Link to="/review"><Button onClick={() => {}} className="buttons-container__add">
+            <Icon type="plus" style={{ color: '#fff' }} />
+            <span className="buttons-container__add__text">
+              Add your recommendation
+            </span>
+          </Button></Link>
+        </div>
+        <p className="heading-name-location">{`${name}, ${country}, ${city}`} </p>
+        <a href={website} className="heading-website">{website}</a>
+        <div className="singlePlace-slider">
```

Comment:
```suggestion
        <div className="single-place__slider">
```
respectively css changes